File: backend/src/main/java/com/example/SEPDrive/controller/kundeDTO.java
package com.example.SEPDrive.controller;

import com.example.SEPDrive.model.carClass;
import com.example.SEPDrive.model.role;
import jakarta.annotation.Nullable;

import java.time.LocalDate;

public record kundeDTO(
            Integer id,
            String userName,
            String email,
            String firstName,
            String lastName,
            LocalDate dateOfBirth,
            byte[] profilePhoto,
            String dtype,
            String carClass
)
{ }

File: frontend/Angular/src/app/models/kunde-dto.ts
export interface kundeDto {

  id: number;
  profilePhoto?: FormData;
  userName?: string;
  firstName?: string;
  lastName?: string;
  email?: string;
  dateOfBirth?: Date;
  password?: string;
  dtype : 'Kunde' ;
  carClass?: string;
  online?: true;
  lastSeen?: '213-05-2025';

}
